dp/generate_prsn.py

```diff
@@ -166,8 +167,14 @@ def check_pr_units(pr_units):
     valid_units = [
         Unit('kg / m**2 / s'),
         Unit('mm / s'),
+	Unit('mm / d'),
         Unit('kg / d / m**2'),
-        Unit('kg / m**2 / d')
+        Unit('kg / m**2 / d'),
+	Unit('g / cm**2 / s'),
+	Unit('cm / s'),
+	Unit('cm / d'),
+	Unit('g / d / cm**2'),
+	Unit('g / cm**2 / d')
     ]
```

Comment:
It may be helpful to order these in some manner (maybe grouping by `mm`, `cm`, `g`, `kg`) just to make it easier to follow.  You may want to double check with @corviday to ensure we aren't missing any cases here.